app/src/main/kotlin/com/aliucord/manager/ui/screens/plugins/model/PluginManifest.kt

```diff
@@ -20,7 +20,9 @@ data class PluginManifest(
     val repositoryUrl: String
         get() = updateUrl
             .replace("raw.githubusercontent.com", "github.com")
+            .replace("cdn.jsdelivr.net/gh/", "github.com/")
             .replaceFirst("/builds.*".toRegex(), "")
+            .replaceFirst("@.*".toRegex(), "")
```

Comment:
This seems like a very flimsy replace, I would rather the entire url be replaced all at once, matching something like: `^https?://cdn.jsdelivr.net/gh/(\w+?)/(\w+?)@(.+?)$`